book/_toc.yml

```diff
@@ -42,16 +40,16 @@
             - file: tutorials/database/3_get_spiral_example
             - file: tutorials/database/4_plot_raster_example
         - file: tutorials/microstructure/microstructure-tutorial
-
+        - file: tutorials/camera-traps-tutorial/camera-traps-notebook
 - part: Projects
   chapters:
     - file: projects/index
       sections:
         - file: projects/project_initialization
         - file: projects/project_roles
         - file: projects/example_workflow
-
 - part: Reference
   chapters:
     - file: glossary
     - file: bibliography
+ >>>> main
```

Comment:
you'll want to delete this line (it's left over from resolving the merge conflict)